app/src/main/java/wishcantw/vocabulazy/activities/player/view/PlayerOptionView.java

```diff
@@ -154,9 +152,9 @@ public void setOptionInModeContent(OptionSettings option, boolean init) {
         mListOrderRadioGroup.check(listOrderIdx);
         mVocOrderRadioGroup.check(vocOrderIdx);
 
-        mPlayerOptionSeekBarsView.setSeekBarProgress(IDX_SEEK_BAR_REPEAT, option.getItemLoop());
+        mPlayerOptionSeekBarsView.setSeekBarProgress(IDX_SEEK_BAR_REPEAT, option.getItemLoop()-1);
         mPlayerOptionSeekBarsView.setSeekBarProgress(IDX_SEEK_BAR_SPEED, option.getSpeed());
-        mPlayerOptionSeekBarsView.setSeekBarProgress(IDX_SEEK_BAR_PLAY_TIME, option.getPlayTime());
+        mPlayerOptionSeekBarsView.setSeekBarProgress(IDX_SEEK_BAR_PLAY_TIME, option.getPlayTime()-10);
```

Comment:
It makes sense to me. The logic is definitely right.
But one more question, so in play time case, the user can only listen up to 40 minutes and must be stopped after that? Just an using circumstance discussion.